scipy/stats/tests/test_mstats_basic.py

```diff
@@ -1355,11 +1355,7 @@ def test_skewtest(self):
                 r = stats.skewtest(x)
                 rm = stats.mstats.skewtest(xm)
                 assert_allclose(r[0], rm[0], rtol=1e-15)
-                # TODO this test is not performed as it is a known issue that
-                # mstats returns a slightly different p-value what is a bit
-                # strange is that other tests like test_maskedarray_input don't
-                # fail!
-                #~ assert_almost_equal(r[1], rm[1])
+                assert_almost_equal(r[1], rm[1])
```

Comment:
maybe swap these to `assert_allclose`?
re:
https://numpy.org/devdocs/reference/generated/numpy.testing.assert_almost_equal.html
seems is encouraged to use allclose instead.

Reply:
Done